infra/main.bicep

```diff
@@ -59,17 +70,27 @@ module containerApps 'core/host/container-apps.bicep' = {
   }
 }
 
-// PostgreSQL Server
-module postgresServer 'db.bicep' = {
-  name: 'postgres-server'
+// Admin app
+module admin 'admin.bicep' = {
+  name: 'admin'
   scope: resourceGroup
   params: {
-    name: '${prefix}-postgresql'
+    name: '${prefix}-admin'
     location: location
     tags: tags
-    postgresAdminUser: postgresAdminUser
-    postgresAdminPassword: postgresAdminPassword
-    postgresDatabaseName: postgresDatabaseName
+    identityName: '${prefix}-id-admin'
+    containerAppsEnvironmentName: containerApps.outputs.environmentName
+    containerRegistryName: containerApps.outputs.registryName
+    exists: adminAppExists
+    postgresServer: '${prefix}-postgresql.postgres.database.azure.com'
```

Comment:
We should find a way that we can leverage the outputs from the `db.bicep` module. As mentioned in another comment, we can probably do this by splitting the server creation and the DB script execution into two separate modules.